### ISSUE TITLE
One button Timing..suggestion

### ISSUE THREAD

**Issue by GoogleCodeExporter** [quemoni/subtitleedit#110]
```
It would be great if there could be one button that when pressed entered the 
end time and auto filled the start time of the next line  based on the end time 
of the previous lines out time.

Thanks..


```

Original issue reported on code.google.com by `leeerick...@gmail.com` on 28 Sep 2012 at 2:46

**Comment by GoogleCodeExporter** [comment]
```
I presume you're talking about "Create mode" ?
```

Original comment by `nikse.dk@gmail.com` on 28 Sep 2012 at 9:31

**Comment by GoogleCodeExporter** [comment]
```
This shortcut is available in create mode in 3.3 (beta)
```

Original comment by `nikse.dk@gmail.com` on 1 Nov 2012 at 5:07
- Changed state: **Fixed**
